### ISSUE TITLE
Minifier html

### ISSUE THREAD

**Issue by oscarlosan** [flightphp/core#549]
Hello, before I used the following function to minify my html and it worked for me. With flightphp it does not minimize the html. Is there something stopping ob_start?

`<?php
ob_start("minifier");
function minifier($code) {
    $search = array(
         
        // Remove whitespaces after tags
        '/\>[^\S ]+/s',
         
        // Remove whitespaces before tags
        '/[^\S ]+\</s',
         
        // Remove multiple whitespace sequences
        '/(\s)+/s',
         
        // Removes comments
        '//'
    );
    $replace = array('>', '<', '\\1');
    $code = preg_replace($search, $replace, $code);
    return $code;
}`

**Comment by n0nag0n** [comment]
With the 3.5 release, we had to overhaul how the output buffer works. You have a couple options, some simpler than others to get it working again.

1. `Flight::set('flight.v2.output_buffering', true);` https://docs.flightphp.com/learn/migrating-to-v3 - This will implement the default behavior you're likely used to.
2. All the output is now written to the `Flight::response()->body` property. Although as I write this out, there isn't a way to completely overwrite the body without calling `Flight::response()->clear()` and then running your minified function on the existing `body` content.

As a side note, I didn't see you close the output buffer, which might be part of the issue? Usually there's something like a ob_end_clean(); to properly close the output buffer (which was part of the problem that was addressed in the 3.5 update).
